sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -2,11 +2,12 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
-from typing import Any, Union, TYPE_CHECKING
+from typing import Any, Dict, Union, Optional, TYPE_CHECKING
 import logging
 from weakref import WeakSet
 
 import uamqp
+from uamqp.constants import TransportType
```

Comment:
I think this line is correlated with "transport_type: Optional[TransportType] = TransportType.Amqp"

however, we should not leak uamqp types, we should use `~azure.servicebus.TransportType`